scripts/utilities/getPackagePath/index.js

```diff
@@ -6,12 +6,8 @@ module.exports = packageName => {
 
   const match = getPackages().find(matchesPath);
 
-  // covers case in jenkins where the base level `psammead` folder isnt
-  // simply called `psammead` but instead has a hash on the end
-  if (!match && packageName === 'psammead') {
-    return getPackages().find(
-      packagePath => !packagePath.includes('/packages/'),
-    );
+  if (packageName === 'psammead') {
```

Comment:
This conditional can be at the top of the function now to early-out immediately, and the match variable is no longer needed